image_data_sar/review.py

```diff
@@ -0,0 +1,204 @@
+import re
+from itertools import count
+import pickle
+import gzip
+import numpy as np
+import astropy.units as u
+
+from cxotime import CxoTime
+from starcheck.parser import get_cat
+from Chandra.Maneuver import duration
+
+
+def get_options():
+    import argparse
+    parser = argparse.ArgumentParser(
+        description="Image data SAR tool")
+    parser.add_argument('pkl',
+                        type=str,
+                        help="Proseco pkl")
+    parser.add_argument('starcheck',
+                        type=str,
+                        help="Starcheck txt file")
+    parser.add_argument('--sched-stop',
+                        type=str,
+                        help="Schedule stop time")
+    args = parser.parse_args()
+    return args
+
+
+def custom_read_starcheck(starcheck_file):
+    sc_text = open(starcheck_file, 'r').read()
+    chunks = re.split(r"={20,}\s?\n?\n", sc_text)
+    catalogs = []
+    for chunk, idx in zip(chunks, count()):
+        obs = get_cat(chunk)
+        if obs:
+            obs['orig_text'] = chunk
+            catalogs.append(obs)
+    return catalogs
+
+
+def starcheck_for_obsid(obsid, sc):
+    for cat in sc:
+        if cat['obsid'] == int(obsid):
+            return cat
+
+
+def obs1_setup_ok(sc_cat):
```

Comment:
If it is just dither and size, this could be generalized, but I wasn't sure if explicit was just better for this kind of quick use case.